src/functions/createCategory/schema.ts

```diff
@@ -0,0 +1,23 @@
+export default {
+    type: 'object',
+    properties: {
+        name: { type: 'string' },
+        description: { type: 'string' },
+        macroCategorieId: { type: 'string' },
+        taxes: { type: 'number' },
+        customSpecsTemplate: {
+            type: 'array',
+            items: {
+                type: 'object',
+                properties: {
+                    fieldName: { type: 'string' },
+                    unitMisure: { type: 'string' }
+                },
+                required: ['fieldName', 'unitMisure', 'value']
```

Comment:
I thinjìk required files is only `fieldName`. There's no `value` and `unitMisure` is optional.